sites/skeleton.dev/src/routes/(inner)/components/autocomplete/+page.svelte

```diff
@@ -39,15 +39,15 @@
 
 	let inputDemo = '';
 	let inputAllowlist = '';
-	type FlavorOption = AutocompleteOption<string>;
-	const flavorOptions = [
```

Comment:
Had to change this because otherwise TS would infer `Meta` generic type as `{healthy: false}` instead of `boolean` and then would error that other meta tags do not satisfy the type.